keps/sig-storage/1790-recover-resize-failure/README.md

```diff
@@ -288,73 +287,70 @@ after expansion is complete even with older kubelets. No recovery from expansion
 
 ### Rollout, Upgrade and Rollback Planning
 
-_This section must be completed when targeting beta graduation to a release._
-
 * **How can a rollout fail? Can it impact already running workloads?**
   This change should not impact existing workloads and requires user interaction via reducing pvc capacity.
 
 * **What specific metrics should inform a rollback?**
+  No specific metric but if expansion of PVCs are being stuck (can be verified from `pvc.Status.Conditions`)
```

Comment:
In volume expansion PRR, I saw metrics related to errors during certain operations.  This seems like a good spot to have similar metrics.

Reply:
I mentioned a metric for counting expansion failures. The operation duration for recovery is not going to be separate from general expansion but counting expansion successes and failures is useful and hence I have include it.